doc/source/drivers/vector/osm.rst

```diff
@@ -275,6 +275,57 @@ The following open options are supported:
 
       Format for all_tags/other_tags fields.
 
+Examples
+--------
+
+List the contents of a local PBF file, with the original downloaded from https://download.geofabrik.de/europe/bosnia-herzegovina-latest.osm.pbf.
+
+.. tabs::
+
+   .. code-tab:: bash
+
+        ogrinfo bosnia-herzegovina-latest.osm.pbf
+
+   .. code-tab:: bash gdal CLI
```

Comment:
Oddly, this just renders as "gdal CLI." Should we use "ogr2ogr" and "gdal vector info" for the tab names?

![image](https://github.com/user-attachments/assets/1658c580-c1c9-43f1-b8e5-145b5afb6cf6)

Reply:
Yes, I'm struggling with how best to name the two approaches. I used "gdal CLI" and "Traditional" in https://gdal.org/en/latest/tutorials/vector_geoparquet_tut.html - so I guess we should be consistent across the docs. The syntax is `language title` for the tabs.